ion_phys/common.py

```diff
@@ -183,7 +183,8 @@ def index(self, level, M, *, F=None, MI=None, MJ=None):
     def level(self, state):
         """ Returns the level a state lies in. """
         for level, data in self.levels.items():
-            if state in data.slice():
+            sl = data.slice()
+            if state in range(sl.start, sl.stop):
```

Comment:
I think we should get a version of this merged. Would changing this to 
```python3
if state > min(sl.start, sl.stop) and state < max(sl.start, sl.stop):
```
seems like a reasonable compromise.